docs/node/run-your-node/paratime-node.mdx

```diff
@@ -421,6 +421,25 @@ In case of `bwrap` issues you need to adjust your Seccomp or AppArmor profiles t
   --security-opt seccomp=unconfined \
 ```
 
+### Bubblewrap Fails to Create Temporary Directory
+
+If the `/tmp` directory is not writable by the user running the node, the Bubblewrap sandbox may fail to start the ParaTimes. In the logs you will see errors about creating a temporary directory, like:
```

Comment:
The current chapter is a leftover from gitbook, but please always hard-wrap the content to 80 characters so it's easier to review. (logs below excluded)